spec/controllers/api/v1/groups_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::GroupsController, :type => :controller do
+
+end
```

Comment:
I haven't even looked at the specs yet, they were generated with `rails g rspec:install`